anubis/src/components/TrainIdList.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <div>
+    <input
+      ref="search"
+      class="form-control"
+      id="trainID oder so"
```

Comment:
Needs to be changed to trainID

Reply:
We did not use this component and removed it. :duck: